algorithm_exercises/876. Middle of the Linked List/solution.py

```diff
@@ -0,0 +1,12 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+class Solution:
+    def middleNode(self, head: Optional[ListNode]) -> Optional[ListNode]:
+        curr = curr2 = head
+        while curr2 and curr2.next:
+            curr = curr.next
+            curr2 = curr2.next.next
+        return curr
```

Comment:
Add an empty line at the end of the file.
Also we don't need the Solution class

Reply:
Done